docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
```

Comment:
> To most people, they would view "ev3dev" as the whole installation, and associate "ev3dev version" with the generic software version, similar to a version of iOS or WIndows

I get what you are saying here. But I will argue that this would be analogous to upgrading ev3dev-jessie to ev3dev-stretch. Packages are just "apps" in the iOS or Windows sense. Upgrading the kernel package is like installing Windows updates, not like upgrading Windows itself.